core/src/main/java/nl/nn/adapterframework/webcontrol/api/ShowConfigurationStatus.java

```diff
@@ -510,6 +510,7 @@ private ArrayList<Object> mapAdapterPipes(Adapter adapter) {
 				} else if(sender instanceof ITransactionalStorage) { // in case no message log specified
 					ITransactionalStorage<?> store = (ITransactionalStorage<?>) sender;
 					mapPipeMessageLog(store, pipesInfo);
+					pipesInfo.put("messageStoreSender", true);
```

Comment:
At this stage it is an assumption that the transactionalStorager is part of a messageStoreSender.
It would be better to call this property 'hasTransactionalStorage'.